packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -2187,6 +2194,21 @@ RED.nodes = (function() {
             }
         }
 
+        // Config node can use another config node, must ensure that this other
+        // config node is added before to exists when updating the user list
+        const configNodeFilter = function (node) {
+            let count = 0;
+            if (node._def?.defaults) {
+                for (const def of Object.values(node._def.defaults)) {
+                    if (def.type) {
+                        count++;
+                    }
+                }
+            }
+            return count;
+        };
+        new_nodes.sort((a, b) => configNodeFilter(a) - configNodeFilter(b));
+
```

Comment:
The old import code got this right without doing this step. I'd like to understand how these changes broke it - and if there are other scenarios related.

Reply:
The quick fix is to re-add these lines but with this way the node added to the user list is not the Proxy. So need a new loop after the `addNode` loop (2518 on dev) to replace each node object by its Proxy object.

https://github.com/node-red/node-red/blob/bb01f26f068b8e083e35fdf19dc9b36f8cf67068/packages/node_modules/%40node-red/editor-client/src/js/nodes.js#L2426-L2430

Reply:
Have stepped through the logic here and am okay with this proposal.